app.js

```diff
@@ -8,7 +8,9 @@ function Store(name, min, max, avgSales) {
   this.avgSales = avgSales;
   this.hourlySales = [];
   this.totalSales = 0;
+  this.id = name.replace(' ', '').toLowerCase();
   cookieStores.push(this); //pushes new Store instances into cookieStores array
+  this.generateHourly();
 };
 Store.prototype.randomCustomer = function () {
```

Comment:
Give yourself some space (one line break) between functions. It helps with readability.